dist/image-sequencer-ui.js

```diff
@@ -385,35 +385,40 @@ function DefaultHtmlStepUi(_sequencer, options) {
     step.ui =
       '\
       <div class="container">\
+      <form class="input-form">\
+    <div class="panel panel-default">\
+        <div class="panel-heading">\
+          <h3 class="panel-title">' +  
+            '<span class="toggle">' +step.name + '<span class="caret"></span>\
+          </h3>\
+        </div>\
+    <div class="panel-body cal collapse in">\
     <div class="row step">\
-    <form class="input-form">\
     <div class="col-md-4 details">\
     <h3>\
-    <span class = "toggle">' +step.name + ' <i class="fa fa-caret-up toggleIcon" aria-hidden="true"></i></span>' +
-    '<span class="load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></span>' +
+    <span class="load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></span>' +
     '</h3><div class="cal collapse in"><p><i>"'+
       (step.description || "") +
       '</i></p></div>\
     </div>\
-    </form>\
     <div class="col-md-8 cal collapse in step-column">\
       <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
       <div class="step-image">\
-        <a class="cal collapse in"><img alt="" class="img-thumbnail step-thumbnail"/></a>\
+        <a class="cal collapse in"><img alt="" id="myimg" class="img-thumbnail step-thumbnail"/></a>\
```

Comment:
I think using a bit more descriptive ID names can be great.
This adds to the readability.